apps/storybook/tailwind.config.cjs

```
@@ -0,0 +1,16 @@
+const config = require('config/tailwind.config')
+
+module.exports = config({
+  content: [
+    './.storybook/**/*.{js,ts,jsx,tsx}',
+    './src/**/*.{js,ts,jsx,tsx}',
+    // purge styles from grid library
+    './../../packages/ui/src/**/*.{tsx,ts,js}',
+    './../../packages/ui-patterns/src/**/*.{tsx,ts,js}',
+  ],
+  plugins: [require('@tailwindcss/container-queries')],
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)